deltachat-ios/AppDelegate.swift

```diff
@@ -111,6 +111,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         logger.info("---- foreground ----")
         appIsInForeground = true
         startThreads()
+        let userDefaults = UserDefaults.init(suiteName: Constants.sharedUserDefaults)
+        if userDefaults?.bool(forKey: Constants.hasExtensionAttemptedToSend) ?? false {
+            let nc = NotificationCenter.default
+
+            DispatchQueue.main.async {
+                nc.post(
+                    name: dcNotificationChanged,
+                    object: nil,
+                    userInfo: [
+                        //TODO: Use correct message_id and chat_id
```

Comment:
i think, this is not needed, DC_EVENT_MSGS_CHANGED is also sent from the core without these information, the app has to handle that (an actually does afaik)

Reply:
sending an empty hash table for user info is sufficient, indeed. omitting userInfo completely won't work.